localsettings/core.txt

```diff
@@ -47,3 +47,6 @@ $wgFragmentMode = [ 'html5', 'legacy' ];
 
 // Choose a sensible timezone
 $wgLocaltimezone = 'UTC';
+
+// Enable watchlist expiry feature
+$wgWatchlistExpiry = false;
```

Comment:
The commit summary says 'enable', I guess this should be $wgWatchlistExpiry = true;?

Reply:
oops